tests/core/go_test/BUILD.bazel

```diff
@@ -243,3 +243,40 @@ go_test(
         "@io_bazel_rules_go//go/tools/bazel",
     ],
 )
+
+go_library(
+    name = "stackpath_lib",
+    srcs = ["stackpath_lib.go"],
+    importpath = "stackpath_lib",
+    stackpath = "foo",
+)
+
+go_test(
+    name = "stackpath_test",
+    srcs = [
+        "a/stackpath2_test.go",
+        "stackpath_test.go",
+    ],
+    stackpath = "bar",
+    deps = [
+        ":stackpath_lib",
+    ],
+)
+
+go_library(
+    name = "stackpath_test_lib",
+    srcs = [
+        "a/stackpath2_test.go",
+        "stackpath_test.go",
+    ],
+    importpath = "stackpath_test",
+    stackpath = "bar",
+    deps = [
+        ":stackpath_lib",
+    ],
+)
+
+go_test(
+    name = "stackpath_embed_test",
+    embed = [":stackpath_test_lib"],
+)
```

Comment:
This test setup puzzled me a bit as I don't fully understand what's the differences between `:stackpath_test` and `:stackpath_embed_test`.

I am expecting 2-3 sets of tests to demonstrate:
- Default behaviors when `stackpath` is not set
- `stackpath` is set only in test
- `stackpath` is set in dep's lib
- `stackpath` is set in both dep's lib and test

I would suggest rename the test and add some docs into https://github.com/bazelbuild/rules_go/blob/master/tests/core/go_test/README.rst

Reply:
'stackpath` only affects the package itself. It does not affect or depend on `deps'. But he can come via `embed' (like: `importpath`).